### PR TITLE
Add translations for image alt attributes

### PR DESCRIPTION
**Pull request by Lauritz-Tieste**
## Description
I have now included the previously hard-coded alt attributes in the translation files.

My Bootstrap Academy username: Lauritz

### CODE REVIEW

File: components/form/BuyCoins.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		>
 			<img
 				src="/images/coin.png"
-				alt="morphcoins"
+				:alt="t('AltAttributes.Morphcoins')"
```
Comment (Defelo) [inline components/form/BuyCoins.vue]: The corresponding entries in locales/*.json seem to be missing

File: components/input/Media.vue [hunk 2]
```diff
@@ -25,7 +25,6 @@
 				class="object-cover w-full h-full"
 				v-if="url && !video"
 				:src="url"
-				:alt="alt"
```
Comment (Defelo) [inline components/input/Media.vue]: Why has this been removed?

File: components/skill-tree/Progress.vue [hunk 3]
```diff
@@ -11,7 +11,7 @@
       <img
         :src="image"
         class="h-10 w-10 sm:h-12 sm:w-12 object-cover rounded order-1"
-        alt="skill cover"
+        :alt="t('AltAttributes.SkillCover')"
```
Comment (Defelo) [inline components/skill-tree/Progress.vue:14]: The corresponding entries in locales/*.json seem to be missing

**Review by Defelo — CHANGES_REQUESTED** [review]

File: components/input/Media.vue (hunk 2 above)
Comment (Lauritz-Tieste) [reply components/input/Media.vue]: by accident

**Review by Defelo — APPROVED** [review]

**Pull request merged**